birdhouse/components/cowbird/user_workspace_readme.ipynb

```diff
@@ -0,0 +1,130 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "6eb979d5-fc6f-456b-9931-6209958f4bae",
+   "metadata": {},
+   "source": [
+    "# README\n",
+    "\n",
+    "This is the user workspace for the JupyterLab environment.\n",
+    "\n",
+    "## Home Directory Layout\n",
+    "\n",
+    "```\n",
+    "notebook_dir\n",
+    "├── README.ipynb\n",
+    "├── public\n",
```

Comment:
Can you end each dir here with a slash `/` to immediately differentiate when it's a dir and when it's a file.